core/src/test/java/org/testcontainers/junit/DockerComposeExposedPortTest.java

```diff
@@ -0,0 +1,21 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.pass;
+
+public class DockerComposeExposedPortTest {
+
+    @Rule
```

Comment:
could you please convert it to `try-with-resources`-style and maybe just add to some other Docker Compose test